### DISCUSSION TITLE
broken font-charter

### DISCUSSION THREAD

**Discussion by iwelch** [Homebrew/discussions#1154] [Casks]
```
$ brew install --cask font-charter   
==> Downloading https://practicaltypography.com/fonts/Charter%20200512.zip
#=#=#                                                                         
curl: (22) The requested URL returned error: 404 Not Found
Error: Download failed on Cask 'font-charter' with message: Download failed: https://practicaltypography.com/fonts/Charter%20200512.zip
```

**Comment by gromgit** [comment]
That cask is not maintained by Homebrew, so you'd have to alert the actual maintainer.

FWIW, upstream seems to have issued an updated ZIP, hence the broken link: https://practicaltypography.com/fonts/Charter%20210112.zip

**Reply by gromgit** [reply]
Also, from https://practicaltypography.com/charter.html:

> If you run a recent version of Mac OS, Charter is already included as a system font. So if you only want to use it on your own machine, you don’t need to download this version.

**Comment by iwelch** [comment]
happy to oblige, but I do not know how to find the maintainer.

(and indeed maybe this should not be a cask for brew to begin with.  recent
is probably within the last 10 years or so.  I already am scrounging around
how to coax macos ttc fonts into R.)

--
Ivo Welch ***@***.***)

**Reply by carlocab** [reply]
What is the output of `brew info --cask font-charter`?

**Comment by iwelch** [comment]
```
$ brew info --cask font-charter
font-charter: 200512
https://practicaltypography.com/charter.html
Not installed
From:
https://github.com/Homebrew/homebrew-cask-fonts/blob/HEAD/Casks/font-charter.rb
==> Name
Charter
==> Description
None
==> Artifacts
Charter/Charter/OpenType/Charter Bold Italic.otf (Font)
Charter/Charter/OpenType/Charter Bold.otf (Font)
Charter/Charter/OpenType/Charter Italic.otf (Font)
Charter/Charter/OpenType/Charter Regular.otf (Font)
```

**Reply by carlocab** [reply]
You can submit a pull request to update this font to use the [updated URL](https://practicaltypography.com/fonts/Charter%20210112.zip) at https://github.com/Homebrew/homebrew-cask-fonts.

You may wish to read the [contribution guide](https://github.com/Homebrew/homebrew-cask-fonts/blob/master/CONTRIBUTING.md#how-to-contribute).
